ios/brave-ios/Sources/BraveWallet/WalletConstants.swift

```diff
@@ -95,7 +97,7 @@ public struct WalletConstants {
   ) -> OrderedSet<BraveWallet.CoinType> {
     switch mode {
     case .general:
-      return [.eth, .sol, .fil]
+      return [.eth, .sol, .fil, .btc]
```

Comment:
This **needs** checked with feature flag `BraveCore.FeatureList.kBraveWalletBitcoinFeature.enabled`. May need to check against this flag in other spots too (still reviewing). Bitcoin should **not** be visible anywhere in iOS wallet without the feature flag enabled.